robotbase/src/main/java/com/team2357/lib/arduino/ArduinoSensor.java

```diff
@@ -0,0 +1,39 @@
+package com.team2357.lib.arduino;
+
+public class ArduinoSensor extends ArduinoUSBController{
+
+    private String m_deviceName;
+
+    public ArduinoSensor(String ttyDevice, String deviceName) {
+        super(ttyDevice);
+        
+        m_deviceName = deviceName;
+    }
+
+    public boolean getIntakeValue() {
+        boolean value = false;
+        if (this.isConnected()){
+            value = !this.getDeviceFieldBoolean(m_deviceName, "state");
+        }
+        
+        return value;
+    }
+
+    public boolean getFeederValue() {
+        boolean value = false;
+        if (this.isConnected()){
+            value = !this.getDeviceFieldBoolean(m_deviceName, "state");
```

Comment:
Same here, replace m_deviceName with a constant. Usually we would not include constants in files like these, but this one is already implementation specific, so it does not hurt to directly use constants in here.